get-started/08_using_compose.md

```diff
@@ -17,9 +17,8 @@ So, how do we get started?
 
 ## Install Docker Compose
 
-If you installed Docker Desktop/Toolbox for either Windows or Mac, you already have Docker Compose!
-Play-with-Docker instances already have Docker Compose installed as well. If you are on
-a Linux machine, you will need to [install Docker Compose](../compose/install/index.md).
+If you installed Docker Desktop for Windows, Mac, or Linux you already have Docker Compose!
+Play-with-Docker instances already have Docker Compose installed as well. 
```

Comment:
I think we should keep the reference for how to install the compose plugin manually, and mention that it is required for users who install Docker Engine standalone.

```suggestion
If you installed Docker Desktop for Windows, Mac, or Linux you already have Docker Compose.
Play-with-Docker instances already have Docker Compose installed as well.
Standalone installations of Docker Engine requires Docker Compose to be installed as a separate package, see [Install the Compose plugin](../compose/install/linux.md).
```

Reply:
Ah yes. That's a Monday-morning-vibes mistake from me